package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -141,6 +220,19 @@ define([
       this.__isAutoUpdate = !!value;
     },
 
+    /**
+     * Gets a value that indicates if the view is in a dirty state.
+     *
+     * A view is _dirty_ during the period from when the view's model has changed until the view is updated.
```

Comment:
A view is _dirty_ during the period after the view's model has changed to until the time the view is updated.